src/main/java/com/massivecraft/factions/zcore/fupgrades/FUpgradesGUI.java

```diff
@@ -149,7 +149,7 @@ private void updateChests(Faction faction) {
 		String invName = SavageFactions.plugin.color(SavageFactions.plugin.getConfig().getString("fchest.Inventory-Title"));
 
 		for (Player player : faction.getOnlinePlayers()) {
-			if (player.getInventory().getTitle() != null && player.getInventory().getTitle().equalsIgnoreCase(invName))
+			if (player.getOpenInventory().getTitle().equalsIgnoreCase(invName)) //TODO Check if it's the same as : player.getInventory().getTitle()
```

Comment:
The same thing, make sure this is available in ALL APIs, if you can call getInventory() in spigot 1.14 let it be. As it will at least work across all versions!

Reply:
yes this should definitely be checked and is this not a possible null? can the inventory not be present

Reply:
Yep, player.getOpenInventory() is annotated with @NotNull so we can do this.
And it's the same problematic as above...